fdbserver/Knobs.cpp

```diff
@@ -250,8 +250,9 @@ ServerKnobs::ServerKnobs(bool randomize, ClientKnobs* clientKnobs) {
 	init( START_TRANSACTION_BATCH_INTERVAL_LATENCY_FRACTION,     0.5 );
 	init( START_TRANSACTION_BATCH_INTERVAL_SMOOTHER_ALPHA,       0.1 );
 	init( START_TRANSACTION_BATCH_QUEUE_CHECK_INTERVAL,        0.001 );
-	init( START_TRANSACTION_MAX_TRANSACTIONS_TO_START,         10000 );
-	init( START_TRANSACTION_MAX_BUDGET_SIZE,                      20 ); // Currently set to match CLIENT_KNOBS->MAX_BATCH_SIZE
+	init( START_TRANSACTION_MAX_TRANSACTIONS_TO_START,        500000 );
+	init( START_TRANSACTION_MAX_BUDGET_SIZE,                    1000 ); // Currently set to match CLIENT_KNOBS->MAX_BATCH_SIZE
```

Comment:
A budget of 1000 is pretty big. I can understand why that might be beneficial when batch sizes are 1000, but I wonder if this could also sometimes lead to trouble.